src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.js

```diff
@@ -0,0 +1,187 @@
+
+const COLUMNS_NAME = {
+  PATRON: ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.USER_FIRST_NAME],
+  LOSS_TYPE: ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.LOSS_TYPE],
+  LOSS_DATE: ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.LOSS_DATE],
+  INSTANCE: ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.INSTANCE_TITLE],
+  PERMANENT_ITEM_LOCATION: ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.PERMANENT_ITEM_LOCATION],
+  FEE_FINE_OWNER: ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.FEE_FINE_OWNER],
+  FEE_FINE_TYPE: ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.FEE_FINE_TYPE],
+  ACTION: 'ACTION',
+};
+const visibleColumns = [
+  COLUMNS_NAME.PATRON,
+  COLUMNS_NAME.LOSS_TYPE,
+  COLUMNS_NAME.LOSS_DATE,
+  COLUMNS_NAME.INSTANCE,
+  COLUMNS_NAME.PERMANENT_ITEM_LOCATION,
+  COLUMNS_NAME.FEE_FINE_OWNER,
+  COLUMNS_NAME.FEE_FINE_TYPE,
+  COLUMNS_NAME.ACTION,
+];
+const columnWidths = {
+  [COLUMNS_NAME.PATRON]: { max: 150, },
+  [COLUMNS_NAME.LOSS_TYPE]: { max: 150, },
+  [COLUMNS_NAME.LOSS_DATE]: { max: 100, },
+  [COLUMNS_NAME.INSTANCE]: { max: 300, },
+  [COLUMNS_NAME.PERMANENT_ITEM_LOCATION]: { max: 150, },
+  [COLUMNS_NAME.FEE_FINE_OWNER]: { max: 150, },
+  [COLUMNS_NAME.FEE_FINE_TYPE]: { max: 150, },
+  [COLUMNS_NAME.ACTION]: { max: 50, },
+};
+const columnMapping = {
+  [COLUMNS_NAME.PATRON]: <FormattedMessage id="ui-users.lostItems.list.columnName.patron" />,
+  [COLUMNS_NAME.LOSS_TYPE]: <FormattedMessage id="ui-users.lostItems.list.columnName.lossType" />,
+  [COLUMNS_NAME.LOSS_DATE]: <FormattedMessage id="ui-users.lostItems.list.columnName.dateOfLoss" />,
+  [COLUMNS_NAME.INSTANCE]: <FormattedMessage id="ui-users.lostItems.list.columnName.instance" />,
+  [COLUMNS_NAME.PERMANENT_ITEM_LOCATION]: <FormattedMessage id="ui-users.lostItems.list.columnName.permanentItemLocation" />,
+  [COLUMNS_NAME.FEE_FINE_OWNER]: <FormattedMessage id="ui-users.lostItems.list.columnName.feeFineOwner" />,
+  [COLUMNS_NAME.FEE_FINE_TYPE]: <FormattedMessage id="ui-users.lostItems.list.columnName.feeFineType" />,
+  [COLUMNS_NAME.ACTION]: <FormattedMessage id="ui-users.lostItems.list.columnName.actions" />,
+};
+export const lostItemsListFormatter = {
+  [COLUMNS_NAME.PATRON]: (actualCostRecord) => {
+    const lastName = get(actualCostRecord, ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.USER_LAST_NAME], '');
+    const firstName = get(actualCostRecord, ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.USER_FIRST_NAME], '');
+    const middleName = get(actualCostRecord, ACTUAL_COST_RECORD_FIELD_PATH[ACTUAL_COST_RECORD_FIELD_NAME.USER_MIDDLE_NAME], '');
+    let patronName = '';
+
+    if (lastName) {
+      patronName = patronName.concat(lastName, ', ');
+    }
+    if (firstName) {
+      patronName = patronName.concat(firstName, ' ');
+    }
+    if (middleName) {
+      patronName = patronName.concat(middleName);
+    }
```

Comment:
If there is no `firstName` and `middleName` the result will be "Last_Name, ". 
Is it expected?
Or maybe we can not have a user without at least `firstName`?

Reply:
Fixed